script.skin.helper.service/default.py

```diff
@@ -99,36 +114,50 @@ def __init__(self):
                 if resultAction:
                     if "jsonrpc" in resultAction:
                         xbmc.executeJSONRPC(resultAction)
+                        xbmc.executeJSONRPC(resultAction)
```

Comment:
Yes, will do. When I have some leftover time, I will invest some time in debugging this thing to do a proper bug report. For now the additional line will be removed.